src/docs/patterns/tags.mdx

```diff
@@ -0,0 +1,230 @@
+import DarkTheme from '../../app/assets/design-patterns/tags/dark-theme.svg';
+import YellowTagsDont from '../../app/assets/design-patterns/tags/yellow-tags-dont.svg';
+import YellowTags from '../../app/assets/design-patterns/tags/yellow-tags.svg';
+import LowerCase from '../../app/assets/design-patterns/tags/lower-case.svg';
+import TextOnly from '../../app/assets/design-patterns/tags/text-only.svg';
+import LongWords from '../../app/assets/design-patterns/tags/long-words.svg';
+import I18N from '../../app/assets/design-patterns/tags/i18n.svg';
+import MultiTagSameColor from '../../app/assets/design-patterns/tags/multi-tag-same-color.png';
+import MultiTagShiftingDont from '../../app/assets/design-patterns/tags/multi-tag-tag-shifting-dont.png';
+import MultiTagShiftingDo from '../../app/assets/design-patterns/tags/multi-tag-tag-shifting-do.png';
+import MultiTagInfoLine from '../../app/assets/design-patterns/tags/multi-tag-info-line.png';
+import MultiTagEllipsis from '../../app/assets/design-patterns/tags/multi-tag-ellipsis.png';
+import MultiTagWrap from '../../app/assets/design-patterns/tags/multi-tag-wrap.png';
+import MultiTagWrapInline from '../../app/assets/design-patterns/tags/multi-tag-wrap-inline.png';
+import Spec1 from '../../app/assets/design-patterns/tags/spec-1.svg';
+import Spec2 from '../../app/assets/design-patterns/tags/spec-2.svg';
+
+{/* KEYWORDS: design pattern tag status badge label */}
+
+# Tags
+
+<ImageGrid images={[BannerImage]} regularWidth />
+
+Tags, sometimes known as badges and labels, are UI components to indicate statuses with text labels and status colors.
+
+<TOC
+    anchors={[
+        { title: 'Introduction', hash: '#chips' },
+        { title: 'Tags vs. Chips' },
+        { title: 'Common Use Cases' },
+        { title: 'Variants' },
+        { title: 'Colors' },
+        { title: 'Labels' },
+        { title: 'Text-Only', depth: 1 },
+        { title: 'Short Labels', depth: 1 },
+        { title: 'Multiple Tags' },
+        { title: 'Design Specifications' },
+        { title: 'Developers' },
+    ]}
+/>
+
+# Tags vs. Chips
+
+<ImageGrid
+    images={[TagsNChipsTag, TagsNChipsChips]}
+    captionsUnderImages={[`Tags — static and non-interactive.`, `Chips — contextual and interactive.`]}
+/>
+
+The name "tag" may sometimes be used for another UI component "chip", but they serve different purposes. Tags are static and non-interactive. They are commonly used as status indicators or something to capture users’ attention. Chips are versatile, contextual, and interactive. They are used for filtering or suggested actions. Chips may also contain auxiliary elements like leading icons or trailing remove icons.
+
+This guide deals specifically with the design of tags. For chips, please refer to [Material Design's guidelines](https://m2.material.io/components/chips).
+
+<MaterialDesignDescription title={'Chips'} url={`https://m2.material.io/components/chips.html`} />
```

Comment:
this link to .html is throwing a 404, might want to use this instead - https://m2.material.io/components/chips

Reply:
Good catch! Sorry about that.